### ISSUE TITLE
Setup Fails newest release

### ISSUE THREAD

**Issue by stevenfoong** [ampache/ampache#3197]
new release fresh install of ampache-5.2.0_all.zip
Major issue found

USER table and admin user not created during install.

I believe I am facing the same issue with this [#3129]

But unfortunately I am using share web hosting , I can't modify the DB configuration file.

Is there any other workaround ? If no other choice, then I will have to install verstion 5.1.

**Comment by mitchray** [comment]
Can you also try with the latest release 5.2.1? https://github.com/ampache/ampache/releases/tag/5.2.1

**Comment by lachlan-00** [comment]
i think i know where these are getting messed. default of utf8mb4 is set on columns it doesn't need. i'll be fixing up an sql for that soon. i'll also post one here for 5.2.1 you can use

**Comment by stevenfoong** [comment]
thank you.
5.2.1 still the same , I believe due to the utf8mb4_unicode_ci .

**Comment by lachlan-00** [comment]
> thank you. 5.2.1 still the same , I believe due to the utf8mb4_unicode_ci .

yes, i need to shorten up the columns with mb4 as a lot of them are just ascii enums or mbid's

**Comment by lachlan-00** [comment]
for the user table there is a lot of extra bytes we need to cut out
all of those fields are extremely long. a reasonable username would be no more than maybe 20chars so might be good to cut that down to 64?

**Comment by stevenfoong** [comment]
i am using it as my own personal web music station. So I don't need a very long user name. I believe most of the people are the same , that why the default mode is for personal use.

**Comment by lachlan-00** [comment]
I've done a bit of work on this I'll make sure it's installable for 5.3.0.

Will be working on a new release from next week but so far I've condensed a lot of columns and worked on making smaller indexes and columns while doing multi artist

**Comment by mitchray** [comment]
Closing due to inactivity, reopen if still relevant in version 7+
